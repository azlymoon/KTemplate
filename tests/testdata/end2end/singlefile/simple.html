{{ 10 }}
{% let $null = null %}
{{ $null }}
{% let $s1 = 'aaa' %}
{{ $s1 }}
test_name='{{ test_name }}'
{% let $bool_true = true %}
true->{{ $bool_true }}
{% let $bool_false = false %}
false->{{ $bool_false }}
$s1 == 'aaa'->{{ $s1 == 'aaa' }}
$s1 == ''->{{ $s1 == '' }}
$s1 != 'aaa'->{{ $s1 != 'aaa' }}
$s1 != ''->{{ $s1 != '' }}
