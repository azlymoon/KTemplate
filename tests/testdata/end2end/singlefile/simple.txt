{{ 10 }}
{% let $null = null %}
{{ $null }}
{% let $s1 = 'aaa' %}
{{ $s1 }}
test_name='{{ test_name }}'
{% let $bool_true = true %}
true->{{ $bool_true }}
{% let $bool_false = false %}
false->{{ $bool_false }}
!true->{{ not $bool_true }}
!false->{{ not $bool_false }}
!!true->{{ not not $bool_true }}
!!false->{{ not not $bool_false }}
!!!true->{{ not not not $bool_true }}
!!!false->{{ not not not $bool_false }}
$s1 == 'aaa'->{{ $s1 == 'aaa' }}
$s1 == ''->{{ $s1 == '' }}
$s1 != 'aaa'->{{ $s1 != 'aaa' }}
$s1 != ''->{{ $s1 != '' }}
$s1 . $s->{{ $s1 ~ $s1 }}
$s1 . $s . $s->{{ $s1 ~ $s1 ~ $s1 }}
{% let $i1 = 1 %}{% let $i2 = 2 %}
$i1 > $i2->{{ $i1 > $i2 }}
$i1 < $i2->{{ $i1 < $i2 }}
$i2 > $i1->{{ $i2 > $i1 }}
$i2 < $i1->{{ $i2 < $i1 }}
{{ time.year }} {{ time.month }}
{{ time.year ~ ':' ~ time.month }}
{{ time.year }} {{ time.month }}
{{ time.year ~ ':' ~ time.month }}
{{ x.y.z }}/{{ x.y.z + x.y.z }}
{{ x.y.z + x.y.z }}/{{ x.y.z }}
{{ (x.y.z) }}-{{ (x).y.z }}-{{ (x).(y).(z) }}
{{ ($i2 + $i2) * 6 }}
{{ $i2 + ($i2 * 6) }}
{{ 'a'|strlen }}
{{ ($s1 ~ $s1)|strlen }}
{{ $s1|strlen + 1 }}
{{ 1|add1 }}
{{ 1|add1|add1 }}
{{ 1|add1|add1|add1 }}
{{ 1|sub1|add1 }}
{{ 1|add1|sub1|add1 }}
{{ 1|sub1|sub1|sub1 }}
{{ test_name|strlen }}
{{ test_name|strlen|add1 }}
{{ 'aaaa'|length }}
{{ x.y|length }}
{{ $i1|add(1) }}
{{ 1|add(10)|add1 }}
{{ $i1|add($i2) }}
{{ zero() }}
{{ abs(zero()) }}
{{ abs(zero() - 1) }}
{{ abs(1) }}
{{ fmt('%b', 255) }}
{{ fmt('%b', zero() + 10) }}
{{ concat3('a', 'b', 'c') }}
{{ fmt(concat3('', '%', 'b'), 8) }}
