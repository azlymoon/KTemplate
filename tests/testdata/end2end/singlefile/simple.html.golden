10



aaa
test_name='simple.html'

true->1

false->
$s1 == 'aaa'->1
$s1 == ''->
$s1 != 'aaa'->
$s1 != ''->1
